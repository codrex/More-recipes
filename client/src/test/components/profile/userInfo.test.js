/* eslint-disable react/jsx-filename-extension */
import React from 'react';
import toJson from 'enzyme-to-json';
import UserInfo from '../../../components/pages/Profile/UserInfo';

const props = {
  user: { fullname: 'john doe' },
  editBtnClicked: jest.fn(),
};

describe('UserInfo component ', () => {
  test('expected to render user info component', () => {
    const component = shallow(<UserInfo {...props} />);
    const tree = toJson(component);
    expect(tree).toMatchSnapshot();
  });
});
