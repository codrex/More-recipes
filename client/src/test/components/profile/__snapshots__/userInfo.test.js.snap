// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`UserInfo component  expected to render user info component 1`] = `
<div
  className="info-wrapper
      mr-auto
      d-flex
      flex-column col-xs-12 col-sm-10 col-md-8 col-lg-6"
>
  <div
    className="
        d-flex
        justify-content-around
        flex-column align-items-center"
  >
    <div
      className="avatar avatar-md"
    >
      JD
    </div>
    <h3
      className="text-capitalize s-padding no-margin fullname-text"
    >
      john doe
    </h3>
  </div>
  <div
    className="d-flex no-margin flex-column align-items-center 
        justify-content-center info"
  >
    <li
      className="info-item"
    />
    <li
      className="info-item"
    >
      @undefined
    </li>
  </div>
  <Button
    Tag="button"
    className=" btn-lg center-margin edit-btn btn-secondary"
    dataTarget="#modal"
    dataToggle="modal"
    handleClick={[Function]}
    href=""
    id="editProfile"
    text="edit profile"
    type="button"
  />
</div>
`;
